wcomponents-core/src/main/java/com/github/bordertech/wcomponents/util/StringEscapeHTMLToXML.java

```diff
@@ -0,0 +1,39 @@
+package com.github.bordertech.wcomponents.util;
```

Comment:
Do you think this method could just be added into the HtmlSanitiser util class? It is not really "extending" the apache class.